### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_NumberField_ScreenReader: When the user is trying to increase or decrease the number using the increment or decrement button, there is no screen reader announcement of the value in the field.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#212]
### Actual:
When the user is trying to increase or decrease the number using the increment or decrement button, there is no screen reader announcement of the value in the field.

Similar issue is observed with Jaws

### Expected:
NVDA should be announcing the value of the field when there is increase or decrease in the number using the increment or decrement button.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Jaws;#STP;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10
Screen Reader: NVDA(2021.1), Jaws(2021.2102.34)

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to "Number field: spin button and try to increase the number by hitting increase/decrease button
4. Check the announcement

### User Impact:
Screen Reader users will not get the proper information about increase/decrease in the number

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by fsteffi** [comment]
**GithubTags:**#Closed;
